controllers/apply/awards-for-all/fields.js

```diff
@@ -1394,10 +1514,19 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                 is: 'yes',
                 then: Joi.string()
                     .allow('')
+                    .max(FREE_TEXT_MAXLENGTH.large)
                     .optional(),
                 otherwise: Joi.any().strip()
             }),
-            messages: []
+            messages: [
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `The Beneficiaries specific groups must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
Is this for the 'Other' field on the **Specific groups of people (Step 2 of 9)** page? If so: 

'Other specific groups must be ${FREE_TEXT_MAXLENGTH.large} characters or less`

Reply:
Yep, spot on